src/python/pants/build_graph/target.py

```diff
@@ -45,6 +46,14 @@ def subsystems(cls):
     """
     return tuple()
 
+  @classmethod
+  def alias(cls):
+    """Subclasses should return their desired BUILD file alias.
```

Comment:
This doesn't seem necessary... currently only really used explicitly for macros?